Resources/Maps/omega.yml

```
@@ -56422,12 +57559,22 @@ entities:
       rot: -1.5707963267948966 rad
       pos: -45.5,33.5
       parent: 2
+    - type: DeviceNetwork
+      configurators:
+      - invalid
+      deviceLists:
+      - 24272
```

```suggestion
    - type: DeviceNetwork
      deviceLists:
      - 24272
```
Замени вот так везде где в файле есть `invalid` что бы не фейлить тесты

---

нихуя не понимаю